yb-voyager/cmd/archiveChangesCommand.go

```diff
@@ -47,6 +48,9 @@ Note that: even if some changes are applied to the target databases, they are de
 	Run: archiveChangesCommandFn,
 }
 
+var archivingCopier *EventSegmentCopier
+var archivingDeleter *EventSegmentDeleter
```

Comment:
since they are now moved to global var level. So i wanted their name to be specific instead of a generic name so that to avoid potential duplicacy of var names and bugs in future.

Reply:
For now, I would suggest restoring the names. We can always rename them if such a situation arises in future.